content/glossary/moat/contents.lr

```diff
@@ -0,0 +1,17 @@
+_model: word
+---
+term: moat
+---
+definition:
+
+The interactive tool to get [bridges] from within [Tor Browser].
```

Comment:
Please make the links as [bridges](../bridge), so the links can be translated and reviewed along with the text.

This other way of linking makes it more difficult to review, because the links are farther away. In this case is not so bad, although it contradicts for example the documentation for translators: https://gitlab.torproject.org/tpo/community/l10n/-/wikis/Localization-for-translators#links.